src/usansred/reduce.py

```
@@ -119,6 +148,31 @@ def convert_xy_to_iq(self, xy_data: XYData) -> IQData:
+        are divided by the counts collected at the monitor during such time.
+        """
+        for harmonic in self.detector_data:
+            intensity_normalized = []
+            error_normalized = []
+
+            for monitor_i, monitor_e, detector_i, detector_e in zip(
+                self.monitor_data.iq_data.i,
+                self.monitor_data.iq_data.e,
+                harmonic.iq_data.i,
+                harmonic.iq_data.e,
+            ):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)